AdSecGH/Components/3_Section/EditSection.cs

```diff
@@ -89,6 +98,7 @@ protected override void SolveInstance(IGH_DataAccess DA) {
       } else {
```

Comment:
as above

Reply:
these file are from autoformat, out of scope for this PR